### ISSUE TITLE
Error: View already has a parent. View: Page(13)

### ISSUE THREAD

**Issue by Daxito** [NativeScript/NativeScript#2189]
### Did you verify this is a real problem by searching 
Yes, I saw issues with other controls like GridView or Scrolls but nothing like the issue I have since this is actually related to the view "Page"

### Tell us about the problem
I am using Telerik Analytics and there is this Uncaught Error that is showing in our logs a bunch of times, I am not able to duplicate it since I am not sure how, all I can see is the log and I guess a user whose App keeps crashing.

### Which platform(s) does your issue occur on?
Android

### Please provide the following version numbers that your issue occurs with:
- CLI: 2.0.1
- Cross-platform modules: 2.0.1
- Runtime(s): Android 2.0.0
- Plugin(s): `"lodash": "^4.12.0",
		"moment": "^2.13.0",
		"nativescript-appversion": "^1.2.0",
		"nativescript-dom": "^1.0.3",
		"nativescript-drawingpad": "^1.0.2",
		"nativescript-geolocation": "0.0.11",
		"nativescript-globalevents": "^1.1.0",
		"nativescript-iqkeyboardmanager": "^1.0.1",
		"nativescript-loading-indicator": "^2.0.1",
		"nativescript-master-technology": "^1.0.4",		
		"nativescript-screenshot": "0.0.2",
		"nativescript-snackbar": "^1.0.3",
		"nativescript-sqlite": "^1.0.8",
		"nativescript-telerik-analytics": "^1.0.0",
		"nativescript-telerik-ui-pro": "file:plugins/nativescript-ui-pro.tgz",
		"numeral": "^1.5.3",
		"pubsub-js": "^1.5.3",
		"string": "^3.3.1"`

### Please tell us how to recreate the issue in as much detail as possible.
I don't know how to duplicate this issue, seems like random for the end user, my hope is that by looking at the call stack and the error message you can tell what could be wrong.

### Is there code involved? 

I have the call stack

```
  {
  "nativeException": {    },
  "stackTrace": "java.lang.RuntimeException: Unable to start activity ComponentInfo{com.abc.abc/com.tns.NativeScriptActivity}: com.tns.NativeScriptException: 
Calling js method onCreateView failed\n\nError: View already has a parent. View: Page(13) Parent: Frame(0)\
File: \"/data/data/com.abc.abc/files/app/tns_modules/ui/core/view-common.js, line: 909, column: 12

StackTrace: \n\tFrame: function:'View._addView', file:'/data/data/com.abc.abc/files/app/tns_modules/ui/core/view-common.js', line: 909, column: 19

   Frame: function:'onFragmentShown', file:'/data/data/com.abc.abc/files/app/tns_modules/ui/frame/frame.js', line: 38, column: 11
     Frame: function:'FragmentClass.onCreateView', file:'/data/data/com.abc.abc/files/app/tns_modules/ui/frame/frame.js', line: 538, column: 13


at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2325)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2387)
at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:3947)
at android.app.ActivityThread.access$900(ActivityThread.java:151)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1309)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5254)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)\nCaused by: com.tns.NativeScriptException:
Calling js method onCreateView failed
Error: View already has a parent. View: Page(13) Parent: Frame(0)
File: \"/data/data/com.abc.abc/files/app/tns_modules/ui/core/view-common.js, line: 909, column: 12

StackTrace: \n\tFrame: function:'View._addView', file:'/data/data/com.abc.abc/files/app/tns_modules/ui/core/view-common.js', line: 909, column: 19
Frame: function:'onFragmentShown', file:'/data/data/com.abc.abc/files/app/tns_modules/ui/frame/frame.js', line: 38, column: 11
Frame: function:'FragmentClass.onCreateView', file:'/data/data/com.abc.abc/files/app/tns_modules/ui/frame/frame.js', line: 538, column: 13


at com.tns.Platform.callJSMethodNative(Native Method)\n\tat com.tns.Platform.dispatchCallJSMethodNative(Platform.java:816)
at com.tns.Platform.callJSMethod(Platform.java:715)
at com.tns.Platform.callJSMethod(Platform.java:694)
at com.tns.Platform.callJSMethod(Platform.java:684)
at com.tns.FragmentClass.onCreateView(FragmentClass.java:43)
at android.app.Fragment.performCreateView(Fragment.java:2053)
at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:894)
tat android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1049)
at android.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:1869)
at android.app.Activity.performCreateCommon(Activity.java:5985)
at android.app.Activity.performCreate(Activity.java:5992)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1125)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2278)
.. 11 more
"
}

```

**Comment by NickIliev** [comment]
Hello @Daxito 

I am making a research on the errors that you have provided and about this one I have noticed that it was previously observed here [https://github.com/NativeScript/NativeScript/issues/1415](https://github.com/NativeScript/NativeScript/issues/1415)
It was fixed back then but still you can test your app with **_Developer options -> Don't keep activities option_**  and then try to minimaze & restore the app.

**Comment by Daxito** [comment]
@NickIliev 
By default the "Don't Keep Activities option" is Off, I set it ON just to see what would happen and nothing really happened, the App looks ok. FYI I am not able to duplicate the issue, I just see the logs with those exceptions but no idea how to duplicate.

**Comment by hamorphis** [comment]
I will take a look....

**Comment by hamorphis** [comment]
This issue should be fixed with the upcoming release of version 2.1 at the end of June. Please, re-open this issue in case the problem persists after upgrading to version 2.1.

**Comment by Steviey** [comment]
+1 Version 2.4.2 + RadSideDrawer
